### ISSUE TITLE
Add labels to image of confusion matrix in episode 2

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#368]
The picture of the confusion matrix in episode 2 does not have any labels: https://carpentries-incubator.github.io/deep-learning-intro/2-keras.html#confusion-matrix, whereas the commands in the material actually create these labels.

**Comment by carschno** [comment]
Updated link for Carpentries Lab: https://carpentries-lab.github.io/deep-learning-intro/2-keras.html#confusion-matrix
